README.md

```diff
@@ -134,7 +137,7 @@ If you like this project follow me to know more about me and my future projects.
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
 
-I would like to thank... my coding partners Felipe Haybar and Newton Alhassan.
+I would like to thank... my coding partners Felipe Haybar,Newton Alhassan and Josep L kausi.
 
```

Comment:
Kindly make the following changes to the README.md file:

- [ ] Kindly include the social links of those who contributed to this project in the Author section of your README.md file 👍 
- [ ] In the Future Features section, kindly add 1 - 3 features that can be further implemented in this project. This would make it easier for people to collaborate with you on the project. 👍 
- [ ] Kindly fix the broken `MIT` link that you have at the bottom of the `README.md` file of this project. This is what I see when I click the link:

![image](https://user-images.githubusercontent.com/77106962/205183729-84891f99-8b03-4dec-ba8e-e27d27ca105c.png) 
_Check the comments under the review._

Reply:
Thanks for your feedback. @Gambit142